app/views/devise/registrations/new.html.erb

```diff
@@ -25,10 +25,10 @@
                       required: true,
                       autofocus: true,
                       input_html: { autocomplete: "Email" }%>
-          <%= f.input :country,
+          <%= f.input :country_id,
                       required: true,
                       autofocus: true,
-                      input_html: { autocomplete: "Country" }%>
+                      collection: Country.all %>
```

Comment:
should the collection be sorted? here?

Reply:
Country.all.sort

It'll not be necessary if country class is already ordered
But  I'd put it anyway